### DISCUSSION TITLE
Senic import and duplicates

### DISCUSSION THREAD

**Discussion by Timokirch** [solectrus/solectrus#3034] [Q&A] [answered]
Hi all, 

maybe someone could help me with a basic understanding question: If I run the import of the "historical" csv files - would this create duplicates or would Solectrus detect this? 

Use case: The docker host was down for a few minutes / hours / days / weeks and I want to add the missing data. Do I have to modify the csv files so that these only contain the missing (minutes / hours / days / weeks)?

Kind regards and thanks for your help

**Comment by ledermann** [comment, marked as answer]
Yes, that will work. Importing historical measurements and overwriting existing data is no problem. InfluxDB can handle this (because the values are watts and not watt-hours). There will be no duplicates.
